File: bytes_to_image.js
const fs = require("fs")


const arr = fs.readFileSync("image_bytes.txt", "utf-8").split(",").map(str_num => Number(str_num))

const image_bytes = new Uint8Array(arr);
console.log(image_bytes)

fs.writeFileSync("node_image.png", Buffer.from(image_bytes))

File: bytes_to_image.py
from PIL import Image
import io

arr = []

with open("image_bytes.txt") as file:
    line = file.readline().strip()
    arr_str = line.split(",")
    for item in arr_str:
        arr.append(int(item))


image_bytes = bytes(arr)
img = Image.open(io.BytesIO(image_bytes))
img.show()
img.save("./python_image.png")